src/lib/popup/popup.scss

```diff
@@ -9,3 +9,7 @@
 :host([hidden]) {
   display: none;
 }
+
+:host([constrain-viewport-width]) {
+  max-width: 100vw;
+}
```

Comment:
@jake-crawford-tyler I know your original issue essentially had this specified inside of a media query. I felt that utilizing an attribute for this would be ideal since the select component could set it by default going forward. So basically regardless of viewport size, the `max-width` will be set going forward, and you can now opt-in to allowing your options to wrap. No need to worry about media queries since it will render the same on both desktop and mobile.

Any concerns here from you given your original use case?

Reply:
This sounds ideal actually. I only set the media query as to not collide with any desired look beyond that. Two thumbs up for me.